sudan_health_nutrition_2.R

```diff
@@ -0,0 +1,147 @@
+install.packages("dplyr")
+library(dplyr)
+
+# source from main script
+source("sudan_health_nutrition.R")
+
+# Total number of children that got BCG vaccine
+
+BCG_positive <- sum(child$coverageBCG == 1, na.rm = TRUE)
+print(BCG_positive)
+
+# Total number of children eligible for BCG vaccine
+BCG_eligible <- sum(child$coverageBCG %in% c(0, 1), na.rm = TRUE)
```

Comment:
this is conceptually correct and I think you will get to the answer you need. The other more concise and more coherent way to code this is this way:

```R
BCG_eligible <- sum(!is.na(child$coverageBCG))
```

Here you don't need to put `na.rm = TRUE` because the code will only sum what is not NA.